packages/notification-service/package.json

```diff
@@ -19,6 +19,8 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
+    "@celo/utils": "^0.0.6-beta5",
+    "@celo/walletkit": "^0.0.14",
```

Comment:
Seems like we shouldn't add a dependency on walletkit here given that we want to deprecate that in favor of contract kit anyway. Can you change this?

Reply:
This is to be consistent with the mobile app, which uses `walletkit`: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/exchange/actions.ts#L6
`contractkit` doesn't have the same `getExchangeRate()` wrapper, so using `contractkit` would mean pasting over the code in `walletkit`. 

I created an issue to migrate this and the mobile app away from `walletkit` after the exchange (or another util where this and the mobile app can share logic) is implemented in `contractkit`. https://github.com/celo-org/celo-monorepo/issues/1026

Reply:
Updated ContractKit with the needed functionality
https://github.com/celo-org/celo-monorepo/pull/1083